build.gradle

```diff
@@ -40,8 +40,8 @@ allprojects {
         servoVersion = '0.12.21'
         governatorVersion = '1.17.5'
         archaiusVersion = '0.7.6'
-        jacksonVersion = '2.9.10'
-        jacksonDatabindVersion = '2.9.10.4'
+        jacksonVersion = '2.11.2'
```

Comment:
If this goes through it should require a bump in the eureka minor version from 9 to 10.